---
bool is_success
bool mission_remain
